apps/cli/bin/cli.js

```diff
@@ -1,34 +1,31 @@
 #!/usr/bin/env node
 
+const pkg = require('../package.json');
+const { showErrorMessage, color } = require('../index');
 const open = require('open');
-const { showHelp, showVersion, showErrorMessage } = require('../index');
+const yargs = require('yargs')(process.argv.slice(2))
+  .usage(`Usage: ${pkg.name} [question]`)
+  .example(
+    `npx ${pkg.name} "How can I learn Relay?"`,
+    'Open a tab in your default browser with your doubt tagging the sseraphini',
+  )
+  .version(color(`${pkg.name} ${pkg.version}`, 'green'))
+  .alias('v', 'version')
+  .help('h')
+  .alias('h', 'help');
 
 async function main() {
-  const message = process.argv[2];
-
-  const validOptions = ['-v', '--version', '-h', ' --help'];
-
-  if (message.charAt(0) === '-' && !validOptions.includes(message)) {
-    return showErrorMessage(message);
+  if (Object.keys(yargs.argv)[1] === '$0') {
+    return yargs.showHelp();
```

Comment:
If the IF does `showHelp` and the opposite of the IF does `showErrorMessage`, when did I fall into the `open` method?

![image](https://user-images.githubusercontent.com/5417662/150365290-b3d73cc2-1d36-4dae-b149-579e1211ceeb.png)


/cc @victordantasdev @sibelius

Reply:
Activates the `open` method when the user types a message to be tweeted.  
The first `if` is for when the user just types the name of the program and returns the help message.  
Because the `yargs` returns an object with the parameters passed separating strings from flags and you can test this with `console.log(yargs.argv);`

Reply:
@victordantasdev How can I test it? I couldn't reproduce the open method passing the parameter that should be valid...

![image](https://user-images.githubusercontent.com/5417662/150371500-3f90751d-80df-4e88-9bb2-f78d17bd412f.png)

Reply:
Oops, that's a bug, I'll fix it now.